src/defrag.c

```diff
@@ -398,7 +398,6 @@ static long scanLaterList(robj *ob, unsigned long *cursor, monotime endtime) {
     quicklistNode *node;
     long iterations = 0;
     int bookmark_failed = 0;
-    if (ob->type != OBJ_LIST || ob->encoding != OBJ_ENCODING_QUICKLIST) return 0;
```

Comment:
While you're working on this file, please also fix `activeDefragSdsHashtableCallback()`.  It should be `static`.